contrib/ansible/inventory

```diff
@@ -1,10 +1,9 @@
 
 [masters]
-kube-master-test-01.example.com
+kube-master-test-[1:3].example.com
 
-[etcd]
-kube-master-test-01.example.com
+[etcd:children]
+masters
```

Comment:
This is fine except IMHO it would be good to have the values in the repo here show the concept of installing etcd clusters outside the Kubernetes master nodes.

Then maybe comments in this file leading to the fact you can configure the etcd cluster on the master nodes.